sdk/react/panels/ItemSelectionScrollView.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * Sample React Native App
+ * https://github.com/facebook/react-native
+ */
+'use strict';
+
+import React, { Component } from 'react';
+import {
+  Text,
+  TouchableHighlight,
+  View,
+} from 'react-native';
+
+var Utils = require('../utils');
+var ItemSelectionList = require('./ItemSelectionList');
+var styles = require('../utils').getStyles(require('./style/ItemSelectionScrollViewStyles'));
+
+var ItemSelectionScrollView = React.createClass({
+  propTypes: {
+    items: React.PropTypes.array,
+    selectedItem: React.PropTypes.string,
+    onSelect: React.PropTypes.func,
+    width: React.PropTypes.number,
+    height: React.PropTypes.number,
+    config: React.PropTypes.object
+  },
+
+  isSelected: function(name) {
+    return name && name !== '' && name == this.props.selectedItem;
```

Comment:
can you comment why you are using `==` here instead of `===`, if it is necessary, add a comment in line please.